tests/plugins/uploadwidget/manual/fileformatnotification.md

```diff
@@ -0,0 +1,19 @@
+@bender-tags: 4.19.0, bug, 5095
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, uploadwidget, floatingspace, toolbar, sourcearea
+
+1. Drag and drop, [JPEG2000](../_assets/logo.jp2) into the editor.
```

Comment:
This test works incorrectly on Windows 7 and 10 because it does not recognize `.jp2` files correctly, not providing the browser with the MIME type of the file. Therefore, it's de facto not possible to test on Windows.